platform/darwin/src/MGLMapSnapshotter.mm

```diff
@@ -117,180 +118,190 @@ - (void)startWithQueue:(dispatch_queue_t)queue completionHandler:(MGLMapSnapshot
     
     _loading = true;
```

Comment:
For consistency, get and set `loading` via the property getter and setter instead of digging into the underlying ivar.